src/components/Survey/Survey.js

```diff
@@ -1,50 +1,55 @@
-//this component holds all the behavior cards.
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
 import BehaviorCard from './BehaviorCard';
-import SubmitButton from './SubmitButton';
 
-class Survey extends Component {
+const styles = theme => ({
+    title: {
+        marginLeft: '10vw',
+    },
+});
 
+class Survey extends Component {
     state = {
-        openCard: 0,//this determines which of the cards is being rendered
+        openCard: 0, //this determines which of the cards is being rendered
     }
 
-    componentDidMount() {
+    componentDidMount = () => {
```

Comment:
Lifecycle methods already have the `this` context correctly set, so you don't need to use an arrow function.